test/utils/InstanceTest.sol

```diff
@@ -209,9 +267,14 @@ abstract contract InstanceTest is HyperdriveTest {
 
         // Alice gives approval to the deployer coordinator to fund the market.
         if (asBase && !isBaseETH) {
```

Comment:
finding the branch logic a bit hard to follow at first glance... might be helpful to have some comments around it or restructure